apparmor.d/profiles-s-z/zsys-system-autosnapshot

```diff
@@ -0,0 +1,30 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2022 Jeroen Rijken
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}{s,}bin/zsys-system-autosnapshot
+profile zsys-system-autosnapshot @{exec_path} flags=(complain) {
+  include <abstractions/base>
+
+  @{exec_path}           rm,
+  /{usr/,}bin/{,ba,da}sh rix,
+  /{usr/,}bin/cat        rix,
+  /{usr/,}bin/cp         rix,
+  /{usr/,}bin/rm         rix,
+  /{usr/,}bin/zsysctl    rix,
+  /{usr/,}bin/zsysd      rix,
```

Comment:
Any reason you don't use the new `zsysd` profile in `rPx` here?

Reply:
That would be a typo.